plugins/redis-enterprise/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.radargun</groupId>
+        <artifactId>radargun-plugin-parent</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+
+    <name>Redis Enterprise plugin for RadarGun</name>
+
+    <artifactId>plugin-redis-enterprise</artifactId>
```

Comment:
I don't understand why you're adding a new plugin here, since it seems to be exactly the same as `plugin-redis`. Was it just to add the hack for clearing the `JedisPool`? @rvansa any thoughts on where this could be hooked in instead? I was think that you could stop the client service, but that might effect the benchmark too much.

Reply:
Looking at the code, I'd rather do `<stop /><start />` than hijacking the `clear` method. Feel free to add `RestartStage` or use `<stop restart="true">` if you long for single stage.